spec/models/state_file_base_intake_spec.rb

```diff
@@ -313,4 +313,15 @@
     end
   end
 
+  describe "#eligible_1099rs" do
+    %w[az md nc nj].each do |state_code|
```

Comment:
although nj is not using this method, I wanted to make sure it was documented that they will have it